docs/input/docs/reference/build-servers/gitlab.md

```diff
@@ -13,10 +13,10 @@ A working example of integrating GitVersion with GitLab is maintained in the pro
 
 Here is a summary of what it demonstrated (many more details in the [README.md][])
 
-- Is a working example know as a [Guided Explorations (GE Manifesto)][] - so job logs and package artifacts can be reviewed. The project can also be imported to your own GitLab group or instance.
+- Is a reusable working example known as a Guided Exploration [(GE Manifesto)][] - so job logs and package artifacts can be reviewed. The project can also be imported to your own GitLab group or instance as a starting point for your own work.
 - IMPORTANT: It demonstrates how to override GitLab CI's default cloning behavior so that GitVersion can do a dynamic copy. Selectively clones GitVersion.yml so that these settings take effect. This best practice demonstrates the best way to do this while avoiding a double-cloning of the project (once by GitLab Runner and once by GitVersion). 
-- Implements GitVersion as a CI CD Extension that can be reused across many projects on the same GitLab instance using includes.
-- Implements GitVersion as a single job that runs the GitVersion container and passes the version number downstream into both PIPLINE and JOB level variables, which means...
+- Implements GitVersion as a CI CD Extension that can be reused across many projects using includes.
+- Implements GitVersion as a single job that runs the GitVersion container and passes the version number downstream into both PIPELINE and JOB level variables, which means...
```

Comment:
I just now noticed the use of uppercase here. Are these code references? If so, they should be code formatted by wrapping them in backticks. If not, they can be emphasized by being wrapped in underscores. 

```suggestion
- Implements GitVersion as a single job that runs the GitVersion container and passes the version number downstream into both `PIPELINE` and `JOB` level variables, which means...
```

```suggestion
- Implements GitVersion as a single job that runs the GitVersion container and passes the version number downstream into both _pipeline_ and _job_ level variables, which means...
```